recipes/skinnywms/meta.yaml

```diff
@@ -0,0 +1,57 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "skinnywms" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4796ebd98fe996b761ff197f357943f9a4e377402b414d6cf95c043bf54f3ccd
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:  
+  host:
+    - python
+    - pip
+    - magics
+  run:
+    - python
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - skinnywms
+
+about:
+  home: http://github.com/ecmwf/skinnywms
+  license: Apache
+  license_file: LICENSE.txt
```

Comment:
I opened a PR on this repository to add the license file to the project and also to include it to the manifest, after that you not be necessary to include the license manually with this recipe

Reply:
https://github.com/sylvielamythepaut/skinnywms/pull/1

Reply:
I had put the LICENSE.txt in the recipe/skinnywms .. Is it not the right place ?

Reply:
Yes, it is! :)
But we do that when the project does not ship the license. We try to avoid it as much as possible.
If there is a possibility to ship the license already with your project that would be better if you don't mind.

Reply:
However, for now you can let it as it is, but for the next versions would be good to have the license already shipped with your project
:)

Reply:
Can I put the text in the meta.yaml itself ?

Reply:
What do you mean?